code/modules/virus2/disease2.dm

```diff
@@ -1,4 +1,5 @@
 var/global/list/disease2_list = list()
+var/global/list/virusDB = list()
```

Comment:
Eh OK then.

Hut general rule of thumb: don't keep `/global` for global variables. It doesn't do anything.